### ISSUE TITLE
fgl and fgl-arbitrary need bumps for QuickCheck 2.10 and QuickCheck 2.11

### ISSUE THREAD

**Issue by clinty** [haskell/fgl#74]
```
next goal: fgl-arbitrary (user goal)
rejecting: fgl-arbitrary-0.2.0.3 (conflict: QuickCheck==2.10.1, fgl-arbitrary => QuickCheck>=2.3 && <2.10)
```

**Comment by jchia** [comment]
#75

**Comment by ivan-m** [comment]
Closed in #75
